jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-nondisruptive-ha-template.yaml

```diff
@@ -20,11 +20,12 @@
             TESTHEAD=1
             cloudsource=develcloud{previous_version}
             upgrade_cloudsource=develcloud{version}
-            nodenumber=5
+            nodenumber={nodenumber}
```

Comment:
This is wrong, I actually need nodenumber+1 here (+1 for the lonely node). @jdsn is that possible somehow?